### PR TITLE
Pttb 120 frontend add toggle in billability icon of tracker list

### PR DESCRIPTION
**Pull request by thatguynischal**
It saves the billable value when we toggle it.

### CODE REVIEW

File: src/modules/tracker/components/TrackerList.vue [hunk 1]
```diff
@@ -253,44 +253,52 @@ const handleItemPerPage = (e) => {
...
-                style="font-size: 25px"
-                title="Non-Billable"
-              >
-                <svg
-                  width="25"
-                  height="25"
-                  viewBox="0 0 25 25"
-                  fill="none"
-                  xmlns="http://www.w3.org/2000/svg"
+              <span style="cursor: pointer;" :title="log.billable ? 'Billable' : 'Non billable'"
```
Comment (alex1the1great) [inline src/modules/tracker/components/TrackerList.vue:256]: missing "data-cy" attribute

File: src/modules/tracker/components/TrackerList.vue [hunk 2]
```diff
@@ -253,44 +253,52 @@ const handleItemPerPage = (e) => {
...
+                    log.project.project_id,
+                    log.billable,
+                    log.start_time,
+                    log.end_time,
+                    log.id
+                  )">
+                <p
+                  v-if="getBillable(log.billable) === 'Billable'"
+                  style="font-size: 25px"
+                  class="my-1"
```
Comment (alex1the1great) [inline src/modules/tracker/components/TrackerList.vue:270]: missing "data-cy" attribute

File: src/modules/tracker/components/TrackerList.vue [hunk 3]
```diff
@@ -253,44 +253,52 @@ const handleItemPerPage = (e) => {
...
+                      fill="#FA602D"
+                    />
+                  </svg>
+                </p>
+                <p v-else class="my-1" style="font-size: 25px" @click="() => (log.billable = true)">
```
Comment (alex1the1great) [inline src/modules/tracker/components/TrackerList.vue:287]: missing "data-cy" attribute

**Comment by thatguynischal** [comment]
Data-cy attributes have been added.

**Review by alex1the1great — APPROVED** [review]

**Pull request merged**